app/controllers/concerns/token_authentication.rb

```diff
@@ -34,4 +36,10 @@ def authenticate_request!
     @current_user = User.find_by(id: payload['sub'])
     raise JWT::InvalidPayload, I18n.t('api.core.no_user_mapping') unless current_user
   end
+
+  def verify_token!
```

Comment:
Maybe better call it 'check_token_revocation!' ? Because actual verification happens during token decoding.